modules/core/src/testFixtures/java/org/apache/ignite/internal/testframework/WorkDirectoryExtension.java

```diff
@@ -270,4 +270,15 @@ private static boolean isEmpty(Path dir) throws IOException {
         }
     }
 
+    private static Path getBasePath() {
+        // Maven build folder
+        Path mavenPath = Path.of("target");
```

Comment:
Do you think we need maven?

Reply:
Somebody is still using Maven, I don't want to ruin the Ignite experience for them)